myuw/templates/handlebars/thrive.html

```diff
@@ -3,8 +3,20 @@
     <div class="container what-is-thrive">
         <h2 id="main" class="heading-page">What is Thrive?</h2>
         <div class="thrive-description">
+
+        {{#if target_fyp}}
             <p>Thrive is a series of weekly messages in MyUW designed to help you in your first year. While there are many things to do as you adjust to campus life, these just-in-time tips can help you know when and what to focus on during the coming months. Created by experienced students, faculty, and advisers, the messages provide pointers on how to explore opportunities and make the most of your learning experiences, meet people and connect to important campus resources, and create the foundation of your university career.</p>
 
+        {{else}}
+            <p>The weekly Thrive messages in MyUW are designed to help you adjust to campus life. Thrive messages for transfer students provide timely advice on what to focus on and when during your first quarter. Created by UW students, faculty, and advisers, the messages offer tips on how to
+                <ul>
+                    <li>meet people and connect to important campus resources</li>
+                    <li>explore opportunities and make the most of learning experiences</li>
+                    <li>create a healthy foundation for your UW career</li>
```

Comment:
Also seems weird here that there is no final punctuation for this list, but the next line just starts a sentence. Not sure if you are looking for that type of editing/proof-reading.

Reply:
LOL. I mentioned the same thing to Janice (her copy) but she didn't seem bothered. I'll fix it though.